package.json

```diff
@@ -88,6 +88,7 @@
     "styled-components": "5.3.6",
     "swr": "1.3.0",
     "ts-command-line-args": "^2.3.1",
+    "ts-node": "10.9.1",
```

Comment:
Looks like a change to fixed version and no longer a compatible version dependency. This was probably intentional and or doesn't matter but noting it